identity/identity.go

```diff
@@ -0,0 +1,9 @@
+package identity
+
+type Identity struct {
+	Id string
+}
+
+func NewIdentity(id string) Identity {
```

Comment:
Are we really making "new" identity here or just creating some kind of Identity from string? What about identity.FromString(id string)?

Reply:
Hmm, not sure. The "New" convention is what we usually go for, so maybe worth leaving this as is? Though I do get what you're saying. @tadovas @Waldz

Reply:
new identity created here in manager, and being put in keystore: https://github.com/MysteriumNetwork/node/blob/39cc02af6ff5f79c9803e311cfdce3af4f0f927f/identity/manager.go#L35
I agree with @tadovas

Reply:
to be more specific, I think it goes like `identity.FromEthAddress(address)`

Reply:
Maybe use FromAddress(), with a change of Identity struct Id property to Address?

Reply:
Just need advice on this last bit. Problem with Id -> Address rename is that a lot of outside structures (DTOs) use IdentityId naming.

Reply:
It would make sense if it signature would be FromAddress(ethAddress Address) but now we are "creating" "identity" from string, looks like a simple converter to me, and not to identity but to some kind of Identifier which can be used with identityManager to load real identity

Reply:
Looks like good idea `identity.address`